src/main/resources/application.properties

```diff
@@ -120,3 +120,11 @@ db.numThreads=4
 
 db.skip.liquibase=false
 spring.liquibase.change-log=classpath:/db_scripts/liquibase/db.changelog.yml
+
+menas.keytab=
```

Comment:
I would do it here similar way like for auth.mechanism=<auth mechanism (in-memory/LDAP)>